src/js/dataEntry.js

```diff
@@ -0,0 +1,95 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("gpii.chartAuthoring.dataEntry", {
+        gradeNames: ["gpii.chartAuthoring.templateInjection", "gpii.chartAuthoring.valueBinding", "autoInit"],
+        selectors: {
+            input: ".gpiic-ca-dataEntry-value",
+            percentage: ".gpiic-ca-dataEntry-percentage",
+            label: ".gpiic-ca-dataEntry-label"
+        },
+        strings: {
+            inputPlaceholder: "Value",
+            percentage: "%percentage%"
+        },
+        model: {
+            // value: number
+            label: ""
+            // perecentage: number
+            // total: number
+        },
+        modelRelay: {
+            source: "",
+            target: "percentage",
+            singleTransform: {
```

Comment:
Use a proper transform definition rather than "free" and use the genuine value as "source" - see comment below